examples/cc_anf/pipeline.yml

```diff
@@ -0,0 +1,42 @@
+pr: none
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+    - '*'
+  paths:
+    include:
+      - examples/cc_anf/*
+      - blocks/cycle-install-server-managed-identity.json
+      - blocks/cycle-prereqs-managed-identity.json
+      - blocks/cycle-cli-local.json
+      - blocks/cycle-cli-jumpbox.json
+      - blocks/jumpbox-anf.json.json
```

Comment:
I believe there is an extra `.json` here

Reply:
good catch